### ISSUE TITLE
Modrinth not storing update correctly (0.8.3 -> 0.8.5 & 0.8.5 -> 0/.8.7)

### ISSUE THREAD

**Issue by Witted** [modrinth/code#2384]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)
- [X] I have ensured my Modrinth App installation is up to date

### What version of the Modrinth App are you using?

0.8.7

### What operating systems are you seeing the problem on?

Windows

### Describe the bug

The modrinth application does not track that it has updated correctly. 
Both of the last two version increments have had the same problem.

### Steps to reproduce

Be on 0.8.5.
Run application
It auto-updates to 0.8.7
Close the application
Reopen the application
It auto-updates from 0.8.5 -> 0.8.7 again 

### Expected behavior

The application stores it's version on update so it doesnt update every time the user runs it

**Comment by LiquidLemon** [comment]
I encountered a similar thing on Windows. I had an old installation of 0.6.x and I was auto-updating to 0.10.19. Turns out I actually had two installations of the app (old in Program Files and new in App Data). The auto update was installing in a different place and starting the newest version automatically but then when I closed it and opened it again I was still using the old one, which ended up prompting for the auto-update again. 

Maybe I did something wrong to end up with two installation, not sure. Posting this just in case anybody else has the same issue.

**Comment by Prospector** [comment]
Yeah, some older versions may have ended up installed in the wrong location and you may have two versions on your machine. Make sure to remove the older one and update any shortcuts you have to the latest version.
